### PR TITLE
Add generic message classes for 2.5.1, 2.6, 2.7, 2.7.1, 2.8, and 2.8.1.

### PR DESCRIPTION
**Pull request by Aaron-P**

### CODE REVIEW

**Review by Usualdosage — APPROVED** [review]
Reviewed and approved.

File: NHapi20/NHapi.Base/Model/GenericMessage.cs [hunk 1]
```diff
@@ -66,6 +66,30 @@ public static Type getGenericMessageClass(String version)
 			{
 				c = typeof (V25);
 			}
+			else if (version.Equals("2.5.1"))
```
Comment (milkshakeuk) [inline NHapi20/NHapi.Base/Model/GenericMessage.cs:69]: Would have probably been easier to use a dictionary with the string version as the key and the type as the value.

**Comment by Aaron-P** [comment]
I just continued the pattern that was already there. If they want to
change it to some other structure so be it.

**Comment by milkshakeuk** [comment]
> I just continued the pattern that was already there. If they want to
> change it to some other structure so be it.
> 

@Aaron-P sure, will most likely make the change at some point.

**Pull request merged**